plugins/providers/metabase/client.go

```diff
@@ -133,11 +135,20 @@ func (c *client) GetDatabases() ([]*Database, error) {
 	}
 
 	var databases []*Database
-	if _, err := c.do(req, &databases); err != nil {
-		return nil, err
-	}
+	var response interface{}
+	if _, err := c.do(req, &response); err == nil {
+		if v, ok := response.([]interface{}); ok {
+			err = mapstructure.Decode(v, &databases) // this is for metabase v0.37
+		} else {
+			err = mapstructure.Decode(response.(map[string]interface{})["data"], &databases) // this is for metabase v0.42
```

Comment:
`response.(map[string]interface{})`
need to handle this type assertion to avoid runtime panic